src/lib/network_pool/transaction_pool.ml

```diff
@@ -506,14 +506,6 @@ struct
                   ~initial_minimum_balance))
           |> Option.value ~default:Currency.Balance.zero
 
-    let check_command (t : User_command.t) : User_command.Valid.t option =
```

Comment:
This should probably be fixed up and used again. Calling `add_from_gossip_exn ~verify:(fun _ -> assert false)` is probably not the desired behaviour. See compatible, where we used this function as that `verify` argument.

Reply:
That only when we handle transactions from an accepted block. For the gossiped transactions we call the verifier https://github.com/MinaProtocol/mina/blob/74c9a184cfe69d2af85c3f13226514a3d278f40b/src/lib/network_pool/transaction_pool.ml#L1212